Add a blog post for Jenkins Mirrors IP change (#6483)
jenkins-infra/jenkins.io: content/blog/2023/06/22/2023-06-22-mirrors-jenkins-new-IP.adoc

---
layout: post
title: "New Public IPv4 for Jenkins Mirrors"
tags:
- infrastructure
- mirrors
- jenkins
authors:
- dduportal
opengraph:
  image: /images/post-images/2023/01/12/jenkins-newsletter/infrastructure.png
links:
  discourse: https://community.jenkins.io/t/new-public-ipv4-for-jenkins-mirrors/8137
---

> The Jenkins project provides a download mirror infrastructure allowing to download Jenkins packages and plugins from a download server close to your location.

Reviewer: ```suggestion
The Jenkins project provides a download mirror allowing to download Jenkins packages and plugins from a download server close to your location.
```

Reviewer 2: I would also remove the first "download" to avoid repetition:
```suggestion
The Jenkins project provides a mirror allowing to download Jenkins packages and plugins from a download server close to your location.
```

It provides a "HTTP redirector" service hosted behind the domains `get.jenkins.io`, `mirrors.jenkins.io` and `mirrors.jenkins-ci.org` which has a new public IP since the 12 June 2023.

> Please update your DNS servers and firewall rules to the new IP `20.119.232.75` if you have a restricted environment.

Reviewer 2: ```suggestion
[IMPORTANT]
===============================
Please update your DNS servers and firewall rules to the new `20.119.232.75` IP if you have a restricted environment.
===============================
```
Not sure if this syntax is recognised.

Author: - The suggestion will not work at all, in any cases: https://docs.asciidoctor.org/asciidoc/latest/blocks/admonitions/
- Currently trying on local build to see if the standard Asciidoctor admonition works in the jenkins.io website and how it is rendered: if it works i'll update the PR so we can see on the preview site

Author: It works with the official syntax: updated the PR

> The former redirector service and its previous IPv4 will be decommissioned the 27 of June 2023.

Author: I'm not sure if the `th` isn't a "French speaking English" thing: @MarkEWaite @kmartens27 which would be the best way to write the date here?

> ====

Author: ```suggestion
```

> More details in <https://github.com/jenkins-infra/helpdesk/issues/3351>.

Reviewer: ```suggestion
More details in link:https://github.com/jenkins-infra/helpdesk/issues/3351[the helpdesk issue].
```

Author: I'm not a fan of hidden links by a text: it does not provide any actionnable for the reader and if the generated HTML does not provide the proper ALT and stuff, then it will be hard for users requiring some kind of accssibility (while a full raw URL is always treated properly).
The `link` macro in Asciidoctor is not mandatory with full URL.

However the "chevrons" are an error from me, as asciidoctor does not support it.

Author: ```suggestion
More details in https://github.com/jenkins-infra/helpdesk/issues/3351.
```

---
layout: post
title: "New Public IPv4 for Jenkins Mirrors"
tags:
- infrastructure
- mirrors
- jenkins
authors:
- dduportal
opengraph:
  image: /images/post-images/2023/01/12/jenkins-newsletter/infrastructure.png
links:
  discourse: https://community.jenkins.io/t/new-public-ipv4-for-jenkins-mirrors/8137
---

The Jenkins project provides a download mirror infrastructure allowing to download Jenkins packages and plugins from a download server close to your location.

It provides a "HTTP redirector" service hosted behind the `get.jenkins.io`, `mirrors.jenkins.io` and `mirrors.jenkins-ci.org` domains, with a new public IP (`20.119.232.75`) since the 12th of June 2023.
The former redirector service and its previous IPv4 will be decommissioned the 27 of June 2023.

> [IMPORTANT]
> ====
> Please update your DNS servers and firewall rules to the new IP `20.119.232.75` if you have a restricted environment.

Reviewer 2: ```suggestion
IMPORTANT: Please update your DNS servers and firewall rules to the new IP `20.119.232.75` if you are in a restricted environment.
```
Reading the doc from your link we don't even need blocks as there are only single phrases.

(Also changed the verb)

Author: I have a preference on using the block because the string `IMPORTANT:` does not really make it explicit that an admonition is used, but it is a detail

====

> [TIP]
> ====
> More details in https://github.com/jenkins-infra/helpdesk/issues/3351.
> ====

Reviewer 2: ```suggestion
TIP: More details in https://github.com/jenkins-infra/helpdesk/issues/3351.
```
Same as above.
